.github/workflows/gradle-weekly-tests.yml

```diff
@@ -28,11 +28,8 @@ jobs:
           java-version: 21
           distribution: temurin
 
-      - name: Install Rust
-        uses: actions-rust-lang/setup-rust-toolchain@v1
-
-      - name: Install Corset
-        run: RUSTFLAGS=-Awarnings cargo install --git ssh://git@github.com/ConsenSys/corset --tag v9.7.15 --locked --force
+      - name: Setup Corset
```

Comment:
should you rename to setup rust corset ? (and at the other places too)

Reply:
Done